docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -73,7 +73,7 @@ You will need to repeat these steps if you rotate Teleport's database certificat
 1. Get the Teleport database CRL by running:
 
    ```code
-   $ tctl auth crl --type=db_client > db-ca.crl
+   $ tctl auth crl --type=db-client > db-ca.crl
```

Comment:
I believe `db_client` was correct here. The same goes for `tctl auth rotate`. Only `tctl auto export` uses `db-client` as `type`. @GavinFrazar could you confirm?

Reply:
unfortunately for our UX, that's right: `tctl auth export` uses hyphens, the others do not.